2-datatypes/boolean.js

```diff
@@ -0,0 +1,11 @@
+const prompt = require("prompt");
+
+prompt.start();
+
+prompt.get(["number1", "number2"], main)
+function main(err, res) {
+    const number1 = true(res.number1);
```

Comment:
Please only one input `prompt.get(["input"], main)`.... once u get that you can compare the text as 
```
let parsedInput = false;
if(res.input === "true"){
  parsedInput = true;
} else if (res.input === "false") {
  parsedInput = false;
} else {
  console.log("invalid input");
  return;
}
```

Reply:
sir ,I am blocked.
Can u ping once you are free.